downstream/modules/platform/proc-create-keycloak-realm.adoc

```diff
@@ -44,7 +44,7 @@ spec:
 | metadata.labels.app: |Set labels to a unique value. This is used when creating the client CR.
 | metadata.labels.realm: | Set labels to a unique value. This is used when creating the client CR.
 | spec.realm.id: | Set the realm name and id. These must be the same.
-| spec.realm.realm | Set the realm name and id. These must be the same.
+| spec.realm.realm: | Set the realm name and id. These must be the same.
 | spec.realm.displayname: | Set the name to display.
```

Comment:
Nit that is unrelated to this PR but maybe worth changing. Shouldn't these fields be wrapped in single quotes?

e.g. `spec.realm.realm:`

Reply:
@oraNod I created a Jira issue for the fields format: I'll create a PR for it.